collections/message.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const Message = mongoose.model('Message', {
```

Comment:
Legacy code from David and Jesstern's app. Will remove when rooms are fully implemented